backend/sozisel/lib/sozisel/model/quizzes/quiz_question.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Sozisel.Model.Quizzes.QuizQuestion do
+  use Sozisel.Model.Schema
+  import Ecto.Changeset
+
+  @type t :: %__MODULE__{
+          question: String.t(),
+          answers: [String.t()],
+          correct_answers: [String.t()]
```

Comment:
Right now we are risking that correct answer will not be in answers. Can you please do the inclusion validation in changeset?